packages/host/app/components/ai-assistant/message/index.gts

```diff
@@ -230,6 +271,21 @@ export default class AiAssistantMessage extends Component<Signature> {
         {{/if}}
 
         <div class='content' data-test-ai-message-content>
+          {{#if @reasoningContent}}
+            <div class='reasoning-content'>
+              {{#if (eq 'Thinking...' @reasoningContent)}}
```

Comment:
Can you use `isThinkingMessage` here too?

Reply:
One would think, but it wasn't working, for some reason.